src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala

```diff
@@ -29,7 +29,7 @@ import scala.tools.nsc.util.{ClassPath, ClassRepresentation, EfficientClassPath}
  */
 case class AggregateClassPath(aggregates: Seq[ClassPath]) extends ClassPath {
   override def findClassFile(className: String): Option[AbstractFile] = {
-    val (pkg, simpleClassName) = PackageNameUtils.separatePkgAndClassNames(className)
+    val pkg = PackageNameUtils.separatePkgAndClassNames(className)._1
```

Comment:
A matcher may be slightly more readable.
```suggestion
    val (pkg, _) = PackageNameUtils.separatePkgAndClassNames(className)
```

Reply:
I don't have strong opinion on what is most readable. I see all the styles in this codebase. I'll defer to anyone who cares to either approve or object.

Reply:
Yeah, I agree with you